javascript/first-js-oop-project.md

```diff
@@ -0,0 +1,127 @@
+# Short mini project to prepare apprentices for their first JS OOP project
+
+### Projected Time
+
+Total: 65 min- 80min
+- Lesson: 30 min
+- Independent Practice: 15 min
+- Materials: 25 min
+
+### Prerequisites
+
+- [Java script lesson 7](../javascript/javascript-7.md)
+
+
+### Objectives
+
+- To use the new JS object-oriented programming skills to create a few event objects, push them to an array, then use jQuery to display the list of events to an html page.
+
+### Specific Things To Teach
+
+- Creating event array.
+- Iterating through the array.
+- Displaying array items in HTML page.
+
+### Materials
+
+- [Documentation on each statement of jQuery](http://api.jquery.com/jquery.each/)
+- [Arrays in javascript](https://www.w3schools.com/js/js_arrays.asp)
+- [.html function of jQuery](https://www.w3schools.com/jquery/html_html.asp)
+
+### Lesson
+
+- Create a js file ***index.js***. Starting with the project, first we need to create an empty array.
+```javascript
+  var events = new events();
```

Comment:
you can't call and Event class without making it first.  Please make an event class!